pkgs/development/python-modules/mip/default.nix

```diff
@@ -0,0 +1,90 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, cbc
+, cffi
+, dos2unix
+, fetchPypi
+, matplotlib
+, networkx
+, numpy
+, pytestCheckHook
+, pythonOlder
+, gurobi
+, gurobipy
+# Enable support for the commercial Gurobi solver (requires a license)
+, gurobiSupport ? false
+# If Gurobi has already been installed outside of the Nix store, specify its
+# installation directory here
+, gurobiHome ? null
+}:
+
+buildPythonPackage rec {
+  pname = "mip";
+  version = "1.14.1";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "6efa66e6f529d797dbbff4b0d4bc7bd22840ed51ec494cf0173a050c6f8ec353";
```

Comment:
Can this be a SRI hash?

Reply:
i.e. `sha256-xxx`

Reply:
Sorry, I was not aware that SRI hashes are now the recommended practice, also because the prefetcher does not seem to support that output and I had to pass through the experimental `nix` command in order to convert the hash.

Updating to an SRI hash now.

Reply:
No issue! Thank you!